projects/microphysics/scripts/prognostic_run.py

```diff
@@ -17,26 +18,13 @@
 logging.basicConfig(level=logging.INFO)
 
 
-def get_env(args):
-    env = {}
-    env["TF_MODEL_PATH"] = args.model
-    env["OUTPUT_FREQ_SEC"] = args.output_frequency
-    env["SAVE_ZARR"] = args.save_zarr
-    env["SAVE_TFRECORD"] = args.save_tfrecord
-    env["SAVE_NC"] = args.save_nc
-
-    env = {k: str(v) for k, v in env.items()}
-
-    return env
-
-
 CONFIG_PATH = Path(__file__).parent.parent / "configs" / "default.yaml"
 
 parser = argparse.ArgumentParser()
 parser.add_argument(
     "--model",
     type=str,
-    default="NO_MODEL",
+    default="",
```

Comment:
Is the model kept as command line arg (even though it can be provided in the config yaml) so that automated training/prognostic runs can be chained together?

Reply:
in principle, but we don't actually have these pipeline right now. In practice, it's the most commonly changed argument so it's nice to have a flag for it.